node/sc/subbridge.go

```diff
@@ -521,8 +527,28 @@ func (pm *SubBridge) handle(p BridgePeer) error {
 	}
 }
 
+func (sc *SubBridge) resetBridgeLoop() {
+	ticker := time.NewTicker(resetBridgeCycle)
+	defer ticker.Stop()
+
+	for {
+		select {
+		case <-sc.quitSync:
+			return
+		case <-ticker.C:
+			if atomic.CompareAndSwapInt64(&sc.checkConnection, 1, 0) {
```

Comment:
We don't need CAS anymore? There is no race condition to write it.

Reply:
Atomic operation is necessary here.